CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 - Bump AppCompat to v1.3.0
 - Move online patient lookup behind a feature flag
 - Cache build dependencies and intermediates in CI workflows
+- Bump Fragment to v1.3.5
```

Comment:
We're switching to 1.3.6 stable release. Can you update to change to reflect the same?

Reply:
Ah, I thought I made this change. Good catch 👍🏾